### PR TITLE
Docs: Unify imports in the documentation.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Unify imports in the documentation. Closes https://github.com/cksource/ckeditor5-internal/issues/3117.

---

### Additional information

The other 2 PRs are a part of this issue:
* https://github.com/cksource/ckeditor5-internal/pull/3179
* https://github.com/cksource/collaboration-features/pull/5220

The [other issue](https://github.com/cksource/ckeditor5-internal/issues/3035) was tied to imports too. Because of that, this PR also closes https://github.com/cksource/ckeditor5-internal/issues/3035.

### CODE REVIEW

**Comment by gorzelinski** [comment]
You can shorten and change most of the imports from default to named. But not all - some of them will not work after the change. As I was working on this issue, I found some exceptions:

* Editor builds e.g: `import DecoupledEditor from '@ckeditor/ckeditor5-build-decoupled-document/src/ckeditor';`
* Base classes e.g: `import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';`
* UI element e.g: `import ToolbarLineBreakView from '@ckeditor/ckeditor5-ui/src/toolbar/toolbarlinebreakview';`
* Util functions like: `import { getData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';`

Also, I didn't change some imports on purpose.

* I didn't change them retrospectively in files like `update-to-x.md` or changelog.
* I didn't change the imports in the dos and don'ts sections like working with typescript.
* I didn't change them either in the code style guide.

Of course, considering the number of changes, it's possible I missed something. But keep the above list in mind.

File: docs/installation/plugins/installing-plugins.md [hunk 1]
```diff
@@ -61,24 +60,19 @@ Edit the `src/ckeditor.js` file to add your plugin to the list of plugins which
 // The editor creator to use.
 import ClassicEditorBase from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
```
Comment (apadol-cksource) [inline docs/installation/plugins/installing-plugins.md:61]:
```suggestion
import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
```

File: docs/installation/plugins/installing-plugins.md [hunk 2]
```diff
@@ -61,24 +60,19 @@ Edit the `src/ckeditor.js` file to add your plugin to the list of plugins which
...
+import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import { Heading } from '@ckeditor/ckeditor5-heading';
+import { Image, ImageCaption, ImageStyle, ImageToolbar, ImageUpload } from '@ckeditor/ckeditor5-image';
+import { Link } from '@ckeditor/ckeditor5-link';
+import { List } from '@ckeditor/ckeditor5-list';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+
+import { Alignment } from '@ckeditor/ckeditor5-alignment'; // <--- ADDED
 
 export default class ClassicEditor extends ClassicEditorBase {}
```
Comment (apadol-cksource) [inline docs/installation/plugins/installing-plugins.md:77]: export default class CustomEditor extends ClassicEditor {}

File: docs/_snippets/framework/abbreviation-level-1.js [hunk 3]
```diff
@@ -5,18 +5,17 @@
...
 
-import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
-import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
-import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
-import Heading from '@ckeditor/ckeditor5-heading/src/heading';
-import List from '@ckeditor/ckeditor5-list/src/list';
-import Bold from '@ckeditor/ckeditor5-basic-styles/src/bold';
-import Italic from '@ckeditor/ckeditor5-basic-styles/src/italic';
+import { Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
+import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
```
Comment (apadol-cksource) [inline docs/_snippets/framework/abbreviation-level-1.js:9]: let's keep the editor base first in the list. It applies to all examples. It is extended with the plugins so keeping it in the first place is more intuitive.

**Review by apadol-cksource — CHANGES_REQUESTED** [review]

**Review by apadol-cksource — APPROVED** [review]
LGTM

**Pull request merged**